doc-deploy-changelog/action.yml

```diff
@@ -252,13 +237,20 @@ runs:
         with open(towncrier_config, 'r') as f:
             config = toml.load(f)
             directory = config["tool"]["towncrier"]["directory"]
+            project_name = config["tool"]["towncrier"].get("package")
```

Comment:
Maybe, we also need to specify the `package_dir` as per this documentation: https://towncrier.readthedocs.io/en/stable/configuration.html#extra-top-level-keys-for-python-projects

Reply:
I would say that, in that case, projects should define it themselves -- if they need it. It is not strictly needed for building the towncrier changelog IMO. As long as we pass the package name (even if it is not used) and the version, that's good enough

Reply:
Excellent. Thanks for clarifying this.